_analyzers/token-filters/cjk-width.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: default
+title: CJK width
+parent: Token filters
+nav_order: 40
+---
+
+# CJK width token filter
+
+The CJK Width token filter in OpenSearch normalizes Chinese, Japanese, and Korean (CJK) tokens by converting full-width ASCII Character to their standard (half-width) ASCII equivalents and half-width Katakana characters to their full-width.
+
+ - **Converting full-width ASCII Character**: In CJK texts, ASCII characters (such as letters and numbers) can appear in full-width form, which occupies the space of two half-width characters. Full-width ASCII characters are typically used in East Asian typography to align with the width of CJK characters. However, for the purpose of indexing and searching, these full-width characters need to be normalized to their standard (half-width) ASCII equivalents.
+
+    See following example:
+
+        Full-Width: ＡＢＣＤＥ １２３４５
```

Comment:
I think these are better formatted as a code block and aligned so the letter A is approximately at the same column in both lines (lines 16 and 17)